rel-eng/lib/builder.py

```diff
@@ -0,0 +1,12 @@
+from tito.builder import Builder
+
+
+class KojiContainerBuild(Builder):
+
+    def _get_tgz_name_and_ver(self):
+        """
+        Returns the project name for the .tar.gz to build. Normally this is
+        just the project name, but in the case of Satellite packages it may
```

Comment:
Do we need reference for Satellite here?

Reply:
Satellite is good everywhere. RH Multiplier++